pkg/vectorize/date/date.go

```diff
@@ -22,3 +27,18 @@ func datetimeToDate(xs []types.Datetime, rs []types.Date) []types.Date {
 	}
 	return rs
 }
+
+func dateStringToDate(xs *types.Bytes, ns *nulls.Nulls, rs []types.Date) []types.Date {
+	for i := range xs.Lengths {
+		str := string(xs.Get(int64(i)))
+		d, e := types.ParseDatetime(str, 6)
+		if e != nil {
+			// set null
```

Comment:
What about panic here?

Reply:
for example, date_add("xxxx", interval 1 day), the result will be null

Reply:
when the input args is "xxxx", we can't parse it so we will return null. For example, if the users use add_date(date()) from table. Table have many strings, but one of them is invalid to parse, I think return null will cost resources.